### PR TITLE
Limit furnace sound volume on catch-up smelting

### PR DESCRIPTION
**Pull request by Emojigit**
This PR limits the volume of smelting sound if the furnace smelts multiple items in the same timer step.

This PR is ready for review.

### CODE REVIEW

File: mods/default/furnace.lua [hunk 1]
```diff
@@ -224,6 +223,11 @@ local function furnace_node_timer(pos, elapsed)
 		elapsed = elapsed - el
 	end
 
+	if items_smelt ~= 0 then
```
Comment (appgurueu) [inline mods/default/furnace.lua:226]:
```suggestion
	if items_smelt > 0 then
```

(just a matter of taste, feel free to ignore)

File: mods/default/furnace.lua [hunk 2]
```diff
@@ -224,6 +223,11 @@ local function furnace_node_timer(pos, elapsed)
...
+		-- Play cooling sound
+		minetest.sound_play("default_cool_lava",
+			{ pos = pos, max_hear_distance = 16, gain = 0.07 * math.min(items_smelt, 7) }, true)
```
Comment (appgurueu) [inline mods/default/furnace.lua:229]: Makes sense to not let it get arbitrarily loud; how did you choose this? I assume you roughly went for ~0.5 max gain?

**Review by appgurueu — APPROVED** [review]
Tested with the max. of 7, sounds bearable :+1:

File: mods/default/furnace.lua (hunk 2 above)
Comment (Emojigit) [reply mods/default/furnace.lua:229]: At first, I went for 0.05, realizing that it works but was too quiet, so I increased that to 0.07.

**Pull request merged**